pkg/apis/contrail/v1alpha1/status.go

```diff
@@ -0,0 +1,25 @@
+package v1alpha1
+
+import appsv1 "k8s.io/api/apps/v1"
+
+// Status is the status of the service.
+// +k8s:openapi-gen=true
+type Status struct {
+	Active        bool  `json:"active,omitempty"`
+	Replicas      int32 `json:"replicas,omitempty"`
```

Comment:
NIT: Since we have 2 replica fields, may be we could call this one "intendedReplicas" to avoid confusion.

Reply:
I would leave it as replicas